plugins/inventory/vmware_vm_inventory.py

```diff
@@ -136,6 +136,22 @@
                 - Also, transforms property name to snake case.
             type: bool
             default: False
+        proxy_host:
+          description:
+          - Address of a proxy that will receive all HTTPS requests and relay them.
+          - The format is a hostname or a IP.
+          - This feature depends on a version of pyvmomi greater than v6.7.1.2018.12
```

Comment:
```suggestion
          - This feature depends on a version of pyvmomi>=v6.7.1.2018.12.
```
Just short hand. Ignore if you like.